packages/mgt-components/src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -1313,4 +1329,12 @@ export class MgtPersonCard extends MgtTemplatedComponent {
       }
     }
   }
+
+  private sendQuickMessageOnEnter(e: KeyboardEvent) {
+    if (e) {
+      if (e.code === 'Enter') {
+        this.sendQuickMessage();
+      }
+    }
```

Comment:
Unless we're calling this from somewhere other than the event binding then e will always be defined. 

```suggestion
    if (e.code === 'Enter') {
      this.sendQuickMessage();
    }
```

If not, then this could be an optional chain `e?.code`